ons_alpha/jinja2/templates/pages/article_page.html

```diff
@@ -0,0 +1 @@
+{% extends "templates/bulletin_page.html" %}
```

Comment:
This reference results in an error.
```suggestion
{% extends "templates/pages/bulletin_page.html" %}
```

Reply:
gah, overzealous last minute cleanup. great catch. sorted